docs/working-with-projects/working-with-images/managing-helm-charts.md

```diff
@@ -5,7 +5,9 @@ weight: 95
 
 [Helm](https://helm.sh) is a package manager for [Kubernetes](https://kubernetes.io). Helm uses a packaging format called [charts](https://docs.helm.sh/developing_charts). Since version 1.6.0 Harbor is now a composite cloud-native registry which supports both container image management and Helm charts management. Access to Helm charts in Harbor is controlled by [role-based access controls (RBAC)](https://en.wikipedia.org/wiki/Role-based_access_control) and is restricted by projects.
 
-## Manage Helm Charts via the Harbor Interface
+There are two places to manage helm charts. First one is in the ChartMuseum which is provided by Harbor from version 1.6.0. The second one is in the OCI-compatible registry which is provided by Harbor from version 2.0.0.
```

Comment:
add this line at the end. 
'This means you can now manage Helm charts alongside your container images through the same set of projects and repositories.'